app/controllers/generic_object_controller.rb

```diff
@@ -27,6 +29,13 @@ def self.populate_display_methods(record)
     end
   end
 
+  def create_display_methods
```

Comment:
If I read this right, this will define a method for each and any value of `display` that the browser sends. Is that the intended behavior?

Reply:
The intended behavior is to create a `"display_#{key}"` method on-the-fly.

`key` can be any value in the `generic_object_definition.properties[:associations][key]` that is associated with a valid Model Name.
To give you an example, I have a `generic_object_definition` record with the following keys/associations -
```ruby
{"vms"=>"Vm", "services"=>"Service", "cp"=>"ManageIQ::Providers::CloudManager"} 
```

Reply:
The key `cp` in the above case (which is a good example of a non-standard `display` parameter)  would be sent via the `display` parameter to the `show` method